src/components/PeopleFilters.tsx

```diff
@@ -27,66 +81,40 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
+            {CENTURIES_ARRAY.map(century => (
+              <SearchLink
+                params={getCenturiesParams(century, centuries)}
+                data-cy="century"
+                className={classNames('button', 'mr-1', {
+                  'is-info': centuries.includes(century),
+                })}
+              >
+                {century}
+              </SearchLink>
+            ))}
           </div>
 
           <div className="level-right ml-4">
-            <a
-              data-cy="centuryALL"
-              className="button is-success is-outlined"
-              href="#/people"
+            <SearchLink
+              params={{ centuries: null }}
+              data-cy="century"
+              className={classNames('button', 'mr-1', {
+                'is-info': !centuries.length,
+              })}
             >
               All
-            </a>
+            </SearchLink>
           </div>
         </div>
       </div>
 
       <div className="panel-block">
-        <a
+        <SearchLink
+          params={{ sex: null, query: null, centuries: null }}
```

Comment:
What if you have 10 filters? Would set `null` to all of them? What if you do `params={new URLSearchParams()}`?

Reply:
Out of interest, if we had to reset only filters and not sorting params (that's what I thought had to be done), how would you write it?

Reply:
Oh actually I would create a new URLSearchParams object and I would add there fields that should remain